Makefile

```diff
@@ -849,15 +850,15 @@ manpages: contrib/man dist/nexd dist/nexctl ## Generate manpages in ./contrib/ma
 # Nothing to see here
 .PHONY: cat
 cat:
-	$(CMD_PREFIX) docker run -it --rm --name nyancat 06kellyjac/nyancat
+	$(CMD_PREFIX) docker run -it --rm --name nyancat 06kellyjac/nyancat@sha256:571396596a0b9367abb72c52619abcbc5b31f766720b181040a33aa959ebfba1
 
 .PHONY: docs
 docs: ## Generate docs site into site/ directory
-	$(CMD_PREFIX) docker run --rm -it -v ${PWD}:/docs squidfunk/mkdocs-material build
+	$(CMD_PREFIX) docker run --rm -it -v ${PWD}:/docs squidfunk/mkdocs-material:9.4 build
 
 .PHONY: docs-preview
 docs-preview: ## Generate a live preview of project documentation
-	$(CMD_PREFIX) docker run --rm -it -p 8000:8000 -v ${PWD}:/docs squidfunk/mkdocs-material
+	$(CMD_PREFIX) docker run --rm -it -p 8000:8000 -v ${PWD}:/docs squidfunk/mkdocs-material:9.4
```

Comment:
if we put all of these in a list of variables near the top, it might help us remember to update them as well as make it easy to know we've checked the full list for what needs updating

Reply:
I pushed a change for this